packages/terra-menu/src/MenuItemGroup.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SingleSelectList from 'terra-list/lib/SingleSelectList';
+import 'terra-base/lib/baseStyles';
+
+const propTypes = {
+  /**
+   * Menu.Items to be grouped together
+   */
+  children: PropTypes.node.isRequired,
+
+  /**
+   * Callback function called when selected index changes
+   */
+  onChange: PropTypes.func,
+};
+
+const defaultProps = {
+  children: [],
+};
+
+const childContextTypes = {
+  isGroupItem: PropTypes.bool,
+};
+
+class MenuItemGroup extends React.Component {
+  getChildContext() {
+    return { isGroupItem: true };
+  }
+  render() {
+    const { children, onChange, ...customProps } = this.props;
+
+    return (
+      <SingleSelectList.Item
```

Comment:
Would we want to default the isSelectable behavior to `isSelectable`? It seems the would be the expected behavior of items in menus. @bjankord @mjhenkes